### DISCUSSION TITLE
Why is babylon_dart so amazing!!!

### DISCUSSION THREAD

**Discussion by unicomp21** [Blimster/babylon_dart#16] [Q&A]
@Blimster just want to say thanks!  babylon_dart is amazing!  Such a joy to use, no struggling w/ webpack, just pure bliss.

**Comment by Blimster** [comment]
Thank you, @unicomp21. Glad to see people having fun with my little contribution to the Dart community.

But I'm curious: for what are you babylon_dart?

**Reply by unicomp21** [reply]
I'm slowly writing a procedural generation framework, inspired by Ken Musgrave and Mojoworld.
